src/JewellryStore/Main.java

```diff
@@ -0,0 +1,220 @@
+                        }
+                        System.out.println("-----------------------------------------");
+                        System.out.println("\nChoose a chain length: ");
+                        System.out.println("\n(1) 30cm (Short) (Default)");
+                        System.out.println("(2) 40cm (Medium)");
+                        System.out.println("(3) 50cm (Long)");
+                        System.out.println("\n-----------------------------------------");
+                        System.out.print("Enter an option number: ");
+                        int necklaceSize = Integer.parseInt(keyboard.readLine());
+                        if (necklaceSize == 2){ // Size cm calculation for toString
+                            necklaceSize = 40;
+                        } else if (necklaceSize == 3){
+                            necklaceSize = 50;
+                        } else { // If user input is invalid, automatic size is the default size
+                            necklaceSize = 30;
+                        }
+                        System.out.print("\nHow many would you like to purchase?: ");
+                        int necklaceQuantity = Integer.parseInt(keyboard.readLine());
+                        double necklacePrice = (double)necklaceQuantity * 24.99;
+                        if(necklaceSize == 40){  // Adds $2.99 to the price for every 10cm longer chain
+                            necklacePrice += (2.99 * necklaceQuantity);
+                        } else if(necklaceSize == 50){
+                            necklacePrice += (2.99 * 2 * necklaceQuantity);
+                        }
+                        subtotal += necklacePrice;
+                        Necklace necklaceOrder = new Necklace(necklaceSize, necklaceQuantity, necklacePrice, necklaceMaterial);
+                        System.out.println("-----------------------------------------");
+                        System.out.println("\n" + necklaceOrder.toString() + " added to bag.");
+                        totalQuantity += necklaceQuantity;
+                        totalUniqueProducts += 1;
+                        theOrder = new Order(subtotal, totalQuantity);
+                        shoppingBag.add(necklaceOrder.toString());
+                        System.out.print("\nEnter -1 to return to the homepage: ");
+                        pageNum = Integer.parseInt(keyboard.readLine());
+                    } else if(productsPageNum == 2){
+                        System.out.println("-----------------------------------------");
+                        System.out.println("\nChoose a material for your ring: ");
+                        System.out.println("(1) Sterling Silver (Default)");
+                        System.out.println("(2) Gold");
+                        System.out.println("(3) Rose Gold");
+                        System.out.println("\n-----------------------------------------");
```

Comment:
The printout of options for an object should be encapsulated inside it's class (in this case in the Necklace class) and not hardcoded into the main program